src/CommunityVoices/App/Website/View/DisplayError.php

```diff
@@ -63,6 +63,35 @@ public function getError($request)
 
         $response = new HttpFoundation\Response($presentation->generate($domainXMLElement));
 
+        $this->finalize($response);
+        return $response;
+    }
+    public function getAllErrors($request) {
+
+        $errorsXMLElement = new SimpleXMLElement(
+            $this->transcriber->toXml(
+                $this->apiProvider->getJson('/errors', $request)
+            )
+        );
+        
+        $errorsPackageElement = new Helper\SimpleXMLElementExtension('<package/>');
+
+        $packagedErrors = $errorsPackageElement->addChild('domain');
+        $packagedErrors->adopt($errorsXMLElement);;
+
+        $packagedIdentity = $errorsPackageElement->addChild('identity');
+        $packagedIdentity->adopt($this->identityXMLElement($request));
+
+        $errorsModule = new Component\Presenter('Module/Errors');
+        $errorsModuleXML = $errorsModule->generate($errorsPackageElement);
+
+        $domainXMLElement = new Helper\SimpleXMLElementExtension('<domain/>');
+        $domainXMLElement->addChild('main-pane', $errorsModuleXML);
+        $domainIdentity = $domainXMLElement->addChild('identity');
+        $domainIdentity->adopt($this->identityXMLElement($request));
```

Comment:
This is done also on lines 82-83?

Reply:
Right, this was a little confusing for me. It looks like most website/view files have the same redundancies, adding the identity as a child of both the domain and the package. Do none of the files need this?

Reply:
Oops, I had no idea we were being redundant like that.  I don't really care to figure out if it's needed or not - if your code still works, you are good to go.